randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -1,857 +1,882 @@
-> Door to Lower Mushrooms; Heals? False; Spawn Point
-  * Layers: default
-  * Normal Door to Lower Mushrooms/Door to Elevator to Green Brinstar
-  > Elevator - Elevator to Green Brinstar
-      Trivial
-
-> Elevator - Elevator to Green Brinstar; Heals? False
-  * Layers: default
-  * Teleporter to Brinstar - Green Brinstar Main Shaft
-  * Extra - scan_asset_id: None
-  * Extra - teleporter_instance_id: None
-  > Door to Lower Mushrooms
-      Trivial
-
-----------------
-Elevator to Red Brinstar
-Extra - map_name: 7962A
-Extra - asset_id: 31
-> Door to Crateria Kihunter Room; Heals? False; Spawn Point
-  * Layers: default
-  * Power Bomb Door to Crateria Kihunter Room/Door to Elevator to Red Brinstar
-  > Elevator - Red Brinstar Zoro Room
-      Trivial
-
-> Elevator - Red Brinstar Zoro Room; Heals? False
-  * Layers: default
-  * Teleporter to Brinstar - Red Brinstar Zoro Room
-  * Extra - scan_asset_id: None
-  * Extra - teleporter_instance_id: None
-  > Door to Crateria Kihunter Room
-      Trivial
-
-----------------
-Forgotten Highway Elevator
-Extra - map_name: 794CC
-Extra - asset_id: 32
-> Door to Forgotten Highway Elbow; Heals? False
-  * Layers: default
-  * Normal Door to Forgotten Highway Elbow/Door to Forgotten Highway Elevator
-  > Elevator - Maridia Elevator Room
-      Trivial
-
-> Elevator - Maridia Elevator Room; Heals? False; Spawn Point
-  * Layers: default
-  * Teleporter to Maridia - Maridia Elevator Room
-  * Extra - scan_asset_id: None
-  * Extra - teleporter_instance_id: None
-  > Door to Forgotten Highway Elbow
-      Trivial
-
+----------------
+Landing Site
+(Valid Starting Location)
+Extra - map_name: 791F8
+Extra - asset_id: 0
+Extra - save_index: 0
+> Door to Gauntlet Access; Heals? False
+  * Layers: default
+  * Normal Door to Gauntlet Access/Door to Landing Site
+  > Landing Site Save Station
+      Any of the following:
+          Screw Attack
+          All of the following:
+              Morph Ball
+              Morph Ball Bombs or Power Bombs
+
+> Door to Parlor; Heals? False
+  * Layers: default
+  * Normal Door to Parlor/Door to Landing Site
+  > Landing Site Save Station
+      Trivial
+
+> Door to Crateria Power Bomb Room; Heals? False
+  * Layers: default
+  * Normal Door to Crateria Power Bomb Room/Door to Landing Site
+  > Landing Site Save Station
+      Trivial
+
+> Door to Crateria Tube; Heals? False
+  * Layers: default
+  * Super Missile Door to Crateria Tube/Door to Landing Site
+  > Landing Site Save Station
+      Any of the following:
+          Hi-Jump Boots or Screw Attack or Space Jump or Speed Booster
+          All of the following:
+              Morph Ball
+              Morph Ball Bombs or Power Bombs
+
+> Event - Win Game; Heals? False
+  * Layers: default
+  * Event Defeat Mother Brain
+  > Landing Site Save Station
+      Trivial
+
+> Landing Site Save Station; Heals? True; Spawn Point
+  * Layers: default
+  > Door to Gauntlet Access
+      Any of the following:

```

Comment:
I don't see anything for Space Jump + Bombs/PBs/Screw. Probably also an SJ+Speed way to do it from the bit where you get SEX Master shinespark.